### DISCUSSION TITLE
Welcome to dsl4j Discussions!

### DISCUSSION THREAD

**Discussion by miurahr** [eb4j/dsl4j#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

### Now open discussion forum!

**Comment by miurahr** [comment]
@plotn you can make question on discussion instead of  posted on unrelated issue.
https://github.com/eb4j/dsl4j/issues/42#issuecomment-1048733356

**Reply by plotn** [reply]
Thanks! I never used discussions and newer knew about it )))
